aks/dbTemplate.md

```diff
@@ -0,0 +1,195 @@
+<!--
+Copyright (c) 2021, Oracle and/or its affiliates.
+Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+-->
+
+{% include variables.md %}
+
+# Apply Database ARM Template to existing {{ site.data.var.wlsFullBrandName }}
+
+This page documents how to configure an existing deployment of {{ site.data.var.wlsFullBrandName }} with an existing Azure database using Azure CLI. 
+
+You can invoke the database ARM template to:
+
+  - Create a new datasource connection, you can have multiple datasource connections in your cluster.
+
+  - Update an existing datasource connection
+
+  - Delete an existing datasource connection
+
+## Prerequisites
+
```

Comment:
Need intervening text here, in between two headings.

Reply:
Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb